core/src/main/java/com/graphhopper/routing/util/TagParserManager.java

```diff
@@ -268,20 +312,31 @@ public TagParserManager build() {
             for (AbstractFlagEncoder encoder : flagEncoderMap.values()) {
                 if (encoder instanceof RoadsFlagEncoder) {
                     // TODO Later these EncodedValues can be added independently of RoadsFlagEncoder. Maybe add a foot_access and hgv_access? and remove the others "xy$access"
-                    if (!em.hasEncodedValue("car_access"))
-                        _addEdgeTagParser(new DefaultTagParserFactory().create("car_access"), false);
-                    if (!em.hasEncodedValue("bike_access"))
-                        _addEdgeTagParser(new DefaultTagParserFactory().create("bike_access"), false);
+                    if (!em.hasEncodedValue("car_access")) {
+                        em.addEncodedValue(new SimpleBooleanEncodedValue("car_access", true), false);
+                        _addEdgeTagParser(new DefaultTagParserFactory().create(em, "car_access"));
+                    }
+                    if (!em.hasEncodedValue("bike_access")) {
+                        em.addEncodedValue(new SimpleBooleanEncodedValue("bike_access", true), false);
+                        _addEdgeTagParser(new DefaultTagParserFactory().create(em, "bike_access"));
```

Comment:
Above you created the parser (OSMRoadEnvironmentParser etc) without the factory. Should this be more consistent? We could e.g. create the TagParserFactory once and then use it everywhere.

Reply:
Ah yes, actually we always create the parsers except for these two new ones (car_access and bike_access). I removed the usages of the DefaultTagParserFactory for these two cases for now. `TagParserManager` will be removed or transformed into something different soon anyway. Then I will check if we can use the factory consistently everywhere or something.